src/convert/streams.ts

```diff
@@ -304,6 +305,13 @@ const makeWriteInfoAbsolute =
   });
 
 const existsOrDoesntMatchIgnored =
-  (forceignore: ForceIgnore) =>
-  (writeInfo: WriteInfo): boolean =>
-    existsSync(writeInfo.output) || forceignore.accepts(writeInfo.output);
+  (forceignore: ForceIgnore, logger?: Logger) =>
+  (writeInfo: WriteInfo): boolean => {
+    const result = existsSync(writeInfo.output) || forceignore.accepts(writeInfo.output);
+
+    // For hidden path detection
+    if (writeInfo.output.includes('/.')) {
+      logger?.debug(`Found file in hidden directory: ${writeInfo.output}`);
+    }
```

Comment:
`'/.` could be one match but a file can also be ignored by a pattern in a user's forceignore + defaults:
https://github.com/forcedotcom/source-deploy-retrieve/blob/be8aec6f54f98849fad875b4b7384a6b0cb213d5/src/resolve/forceIgnore.ts#L21

suggestion:
if a file is forceignored let's log `File ${writeInfo.output} was ignored`.